changelog.txt

```diff
@@ -1,3 +1,8 @@
+Version 5.3.0
```

Comment:
We probably need to create a milestone?

Reply:
let's discuss on monday. i think we just run a new build w/5.2.4

Reply:
@jmprieur as discussed, 5.2.4